### DISCUSSION TITLE
Can't correctly pair sonoff snzb-01 to zigbee bridge

### DISCUSSION THREAD

**Discussion by Hoeby** [arendst/Tasmota#15139] [Q&A] [answered]
I am trying to pair 2 sonoff snzb-01 to a flashed sonoff bridge.

I put the bridge in pairing mode. Next i push the reset button for 5 seconds, a red led starts flashen for 3 times. In the console i see communication, it is added to the device list. But when pushing the button, nothing is happening. I see not data in the console, that i have pushed.

I have this problem with both buttons. Tried a new fresh battery and very close to the bridge, without luck.

What is going wrong here?

The messages i see in the console:
```
15:41:35.826 ZIG: {"ZbEZSPReceived":"23000800C90C8A790425004B120004"}
15:41:35.830 ZIG: {"ZbEZSPReceived":"2400C90C8A790425004B12000203FFFF"}
15:41:50.183 ZIG: {"ZbEZSPReceived":"23000801C90C8A790425004B120004"}
15:41:50.648 ZIG: {"ZbEZSPReceived":"2400C90C8A790425004B120001000000"}
15:41:50.651 MQT: tele/tasmota_963592/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B002504798A","ShortAddr":"0x0CC9","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
15:41:51.375 MQT: tele/tasmota_963592/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B002504798A","ShortAddr":"0x0CC9","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
15:41:51.967 MQT: tele/tasmota_963592/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x01"]}}
15:41:52.618 MQT: tele/tasmota_963592/SENSOR = {"ZbReceived":{"0x0CC9":{"Device":"0x0CC9","ModelId":"WB01","Manufacturer":"eWeLink","Endpoint":1,"LinkQuality":126}}}
```

**Comment by sfromis** [comment]
You should get further messages, even before  the button is used. In some cases, pairing may be "incomplete"; I'd suggest trying again to try to get a "full" pairing.

**Comment by Hoeby** [comment, marked as answer]
I already tried 10 maybe 20 times. Even with different batteries or distance to the bridge. Everytime the same respone, without luck

**Reply by sfromis** [reply]
Did a small test, paired a new SNZB-02, same family. It paired right away when having Zigbee Permit Join active, reset the sensor by 5 second button push till getting the led flashes, about 6 cm from the bridge (edge to edge).

Strictly speaking, I'm not sure what messages you "should" get after what you posted (if any), I usually see some auto-bind activity from Tasmota, but that would depend a lot on the type of device.

**Reply by Hoeby** [reply]
Tried it again, because you written you where very close to the bridge, with 6cm. 
Put the bridge in joining mode. Put the switch next to the bridge. Houses hit each other. And pushed the reset button untill red flash. Now there are a lot more messages and pairing is succesfull.

Never had a zigbee device that needed to be that close to pair. Normally it works perfect when i am 1 meter from the bridge, but not with the sonoff switch
